docs/user-guide/performance.rst

```diff
@@ -62,6 +62,43 @@ will be one single chunk for the array::
    >>> z5.chunks
    (10000, 10000)
 
+
+Sharding
+~~~~~~~~
+
+If you have large arrays but need small chunks to efficiently access the data, you can
+use sharding. Sharding provides a mechanism to store multiple chunks in a single
+storage object or file. This can be useful because traditional file systems and object
+storage systems may have issues with many small files.
```

Comment:
```suggestion
storage systems may have issues storing and accessing many files.
```
I dropped "small" because I think the issue is number of files, and not there size? I may be wrong there though...

Reply:
Correct. Having small files can be inefficient, e.g. when the fs block size is larger than the file. But that is orthogonal to the performance.